src/main/java/me/totalfreedom/totalfreedommod/command/Command_gtfo.java

```diff
@@ -54,6 +54,9 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
         // Rollback
         plugin.rb.rollback(player.getName());
```

Comment:
Maybe check if CoreProtect is enabled and use that method otherwise use TFM rollback method.